### ISSUE TITLE
npx mongo-migrate status is not showing any migrations

### ISSUE THREAD

**Issue by Ishan5hrestha** [mycodeself/mongo-migrate-ts#119]
![image](https://github.com/user-attachments/assets/e875aa3d-a7c0-485f-84a9-f8cc0294b69c)

I have created migration scripts inside migration folder and when checking migration status or try to migrate, it keeps showing me that there is no migrations

**Comment by designamx** [comment]
try setting `globPattern` to `*_Migration.ts`

**Comment by mycodeself** [comment]
Yep try what @designamx says or just use the pattern `*.ts`. if I'm not wrong the CWD for the glob pattern is relative to the `migrationsDir`.

**Comment by Ishan5hrestha** [comment]
Thanks, my file is now identified. But now I am getting this error even though i have set my package type as module
![image](https://github.com/user-attachments/assets/15e9493f-e29c-4b2e-9638-6b1bace3f65d)
